### PR TITLE
[feat] Various Odahuflowctl improvement

### LINKED ISSUE #103: Error handler in Odahu-flow CLI
For now, the CLI shows a python stack trace during every exception. To be more user-friendly, the CLI must only write a stack trace when the `--verbose` parameter is enabled.

### LINKED ISSUE #105: Bash\ZSH completion for the Odahu-flow CLI
The click library supports the [bash\zsh completion](https://click.palletsprojects.com/en/7.x/bashcomplete/). We have to check it and provide the documentation for users.

### LINKED ISSUE #125: Streams system logs which are not asked
**Description:**
The odahuflowctl outputs system logs when they are not requested, whereas expected behaviour is not outputting logs if a user doesn't indicate it explicitly.

**Steps to reproduce:**
1. Write some command with odahuflowctl (without option --verbose).

**AR:** during the execution **gives** system logs.
**ER:** during the execution **doesn't give** system logs.

### LINKED ISSUE #137: Update/alter/refactor the docs of odahuflowctl
**Description:**
Amendments to the documentation of odahuflowctl.

**Updates:**
1. "_version_" is a command in odahuflowctl but should be the option.
The command for receiving version should be **odahuflowctl --version**.
1. After call: **odahuflowctl bulk** instead of '_a_' should be '_an_'.
image

### LINKED ISSUE #139: add abbreviations for commands to "--help"
**Description:**
There are abbreviations of command for odahuflowctl but they are not displayed in helper (-- help).

**e.g.**
odahuflowctl ti stands for odahuflowctl toolchain-integration
odahuflowctl conn stands for odahuflowctl connection

**ER:**
Abbr. should be represented in odahuflowctl docs (commands: odahuflowctl --help).

### PR DESCRIPTION
**Pull request by mcdoker18**
Stopped to show an error traceback to user. For now we produce
small message when an exception was occurred.

Any logs, including error level, doesn't output until a user
provides the --verbose flag.

Enabled a shell completion for CLI.

Added aliases for the commands in their documentation.

Removed obsolete the sandbox command.

This closes #139
This closes #137
This closes #105
This closes #103
This closes #125

### CODE REVIEW

File: packages/cli/odahuflow/cli/utils/error_handler.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    Print error message to stdout and exit if any error occurs
+    """
+    try:
+        yield
+    except Exception as e:
+        # If the verbose flag is enabled than the stacktrace will be logged.
+        LOG.exception('Exception occurs during CLI invocation')
+        # This message always appears in stdout.
+        # The verbose flag does not affect this.
+        click.echo(f'Error: {str(e)}')
```
Comment (vlad-tokarev) [inline packages/cli/odahuflow/cli/utils/error_handler.py]: Let's add something like: "For more information rerun command with --verbose flag" ?

File: packages/cli/odahuflow/cli/parsers/completion.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+@click.command()
+@click.argument('shell', required=False, type=click_completion.DocumentedChoice(
+    click_completion.core.shells))
+def completion(shell):
+    """
+    Output odahuflowctl completion code to stdout.\n
+    Load the zsh completion in the current shell:\n
+        source <(kubectl completion zsh)
```
Comment (maxkochubey) [inline packages/cli/odahuflow/cli/parsers/completion.py]: `kubectl`? :)

**Comment by BPylypenko** [comment]
And this also could be improved:
instead of --no-verbose should be --no**n**-verbose
and it would be good to add which is default of them.

e.g.
  --no-verbose /  --non-verbose (default)
image

File: packages/cli/odahuflow/cli/parsers/completion.py (hunk 2 above)
Comment (mcdoker18) [reply packages/cli/odahuflow/cli/parsers/completion.py]: Ahaha, fixed

File: packages/cli/odahuflow/cli/utils/error_handler.py (hunk 1 above)
Comment (mcdoker18) [reply packages/cli/odahuflow/cli/utils/error_handler.py]: Done.
The message must appear only when the flag is disabled. But I can determine the flag only by log level. It's a bit awful.

File: packages/cli/odahuflow/cli/parsers/connection.py [hunk 3]
```diff
@@ -34,7 +34,8 @@
 @click.pass_context
 def connection(ctx: click.core.Context, url: str, token: str):
     """
-    Allow you to perform actions on connections
+    Allow you to perform actions on connections.\n
+    Alias for the command is conf.
```
[the remark this answers is not in the document]
Comment (mcdoker18) [reply packages/cli/odahuflow/cli/parsers/connection.py]: Fixed

File: packages/cli/odahuflow/cli/main.py [hunk 4]
```diff
@@ -27,41 +29,59 @@
...
+    route,
+    template,
+    gppi,
+    toolchain_integration,
+    training,
+    login,
+    logout,
+    version,
+    completion,
+]
```
[the remark this answers is not in the document]
Comment (mcdoker18) [reply packages/cli/odahuflow/cli/main.py]: Noop. By the way, I am not sure that click group object are hashable.

**Pull request merged**